docs_src/microservices/device/details/DeviceCommands.md

```diff
@@ -3,6 +3,32 @@ title: Device Service - Device Commands
 ---
 
 # Device Service - Device Commands
+## Concept
+![image](Device_Command.png)
 
-coming soon
+Device commands instruct your device to take an action. If we want to know the current humidity of your device, we can send a request to core command service, which accesses the database to query information about the device and the device profile to collects the information, and passes the request to the device service, which then passes it on to the device/sensor (south side). Eventually we can get to know the current humidity of your device.
 
+## Example and Types
+
+Device commands specify access to reads and writes for multiple simultaneous device resources. In other words, device commands allow you to ask for multiple pieces of data from a sensor at one time (or set multiple settings at one time). 
+
+In this example, we can request both human and dog counts in one request by establishing a device command that specifies the request for both. 
+
```

Comment:
Add something about resources are also implicit Device Commands for the single resource. Explicit device command are only needed to aggregate resources into a single command.

Reply:
updated